components/script/script_thread.rs

```diff
@@ -331,8 +331,7 @@ pub struct ScriptThread {
     #[no_trace]
     layout_factory: Arc<dyn LayoutFactory>,
 
-    // Mouse down point.
-    // In future, this shall be mouse_down_point for primary button
+    // Mouse down point for primary button
```

Comment:
Two slashes means this isn't a rustdoc annotation. Perhaps it should have three slashes and bit fleshed out a bit?